OpenShift/helm/nexus-iq/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
what's appVersion? might we want that to match the server we're shipping?

Reply:
I don't know if it's used, but to be safe I updated it to use the same version as our docker & backend server version 👍

Reply:
Ooh, that's an interesting experiment I'd probably lose lots of time conducting: "Do we really need each line in this file?"